numpyro/optim.py

```diff
@@ -75,6 +77,27 @@ def __init__(self, *args, **kwargs):
         super(Adam, self).__init__(optimizers.adam, *args, **kwargs)
 
 
+class ClippedAdam(_NumpyroOptim):
+    """
+    :class:`~numpyro.optim.Adam` optimizer with gradient clipping.
+
+    **Reference:**
+
+    `A Method for Stochastic Optimization`, Diederik P. Kingma, Jimmy Ba
+    https://arxiv.org/abs/1412.6980
+    """
+    def __init__(self, *args, **kwargs):
```

Comment:
I think we should expose `clip_norm` here and add docs for it.

Reply:
done.